core/arch/arm/kernel/thread_spmc_a32.S

```diff
@@ -50,8 +50,10 @@ END_FUNC ffa_msg_send_direct_resp
 FUNC thread_std_smc_entry , :
 UNWIND(	.cantunwind)
 
+	push	{r4, r5} /* Pass these following the arm32 calling convention */
```

Comment:
OK, use the same comment as in the other file (either mine or yours it doesn't matter much)